requirements.txt

```diff
@@ -1,5 +1,5 @@
 # To be changed to allennlp>=1.0 once that's released.
-git+https://github.com/allenai/allennlp.git@8bc5da2d7d0050374c78b52adde7149425e3b52a
+git+https://github.com/allenai/allennlp.git@644db4998487580fa054d28c32ad3dd8622ad6cb
```

Comment:
That SHA is going to change again, as we squash merge all PRs to allennlp.

Reply:
Yes. And it doesn't work like this because it should say `matt-gardner`, not `allenai`. So I guess we'll have to wait.

Reply:
Actually, it merged just now.

Reply:
It's merged now.  I need to finish an ACL review; feel free to update this and merge yourself when you're happy with it.  Not sure if I'll get back to github today.